roles/preflight/tasks/test_preflight_check_container_one_image.yml

```
@@ -101,6 +101,43 @@
+          vars:
+            filter_params: "filter=docker_image_digest%3D%3D{{ sha.stdout }}"
+          ansible.builtin.uri:
+            url: >
+              {{ catalog_url }}/images?{{ filter_params }}&page_size=100&page=0
+            method: GET
+            headers:
+              X-API-KEY: "{{ PYXIS_API_TOKEN }}"
+            status_code: 200
+            timeout: 120
+          register: pyxis_grade_status
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)